ui/lib/core/addon/components/kv-suggestion-input.hbs

```diff
@@ -0,0 +1,31 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+<div ...attributes {{did-update this.updateSuggestions @mountPath}}>
+  <FormFieldLabel @label={{@label}} @subText={{@subText}} />
```

Comment:
Do we want to wrap the label component in `{{#if @label}}` for flexibility? That could also be a future change when this is implemented elsewhere

Reply:
Good idea! I'll go ahead and do it now since it's an easy update.